hebsafeharbor/identifier/signals/hebrew_city_recognizer.py

```diff
@@ -13,7 +13,8 @@ class AmbiguousHebrewCityRecognizer(LexiconBasedRecognizer):
     """
 
     DEFAULT_CONFIDENCE_LEVEL = 0.2  # expected confidence level for this recognizer
-    LOCATION_OVERLAP_FACTOR = 0.8  # enhancement factor for this recognizer if overlap with another geo entity found
+    LOCATION_OVERLAP_FACTOR = 0.4  # enhancement factor for this recognizer if overlap with another geo entity found
+    CONTEXT_ENHANCEMENT_FACTOR = 0.4  # enhancement factor for this recognizer if supportive context found
 
     def __init__(self, name: str, supported_entity: str, phrase_list: List[str], supported_language: str = "he",
                  endorsing_entities=List[str], allowed_prepositions=[], context=List[str]):
```

Comment:
```suggestion
                 endorsing_entities:List[str]=None, allowed_prepositions: List[str]=None, context:List[str] = None):
```

Reply:
Passing a mutable object (an empty list) is not recommended in Python, as changing it in one place would cause a side effect in another. https://docs.python-guide.org/writing/gotchas/#mutable-default-arguments